### DISCUSSION TITLE
Card height not correct when full width

### DISCUSSION THREAD

**Discussion by RhysLees** [laravel/nova-issues#4160] [Ideas & Feature Requests] [answered]
- Laravel Version: 9.x
- Nova Version: 4.3.3
- PHP Version: 8.1
- Database Driver & Version: N/A
- Operating System and Version: Win 11 WSL2
- Browser type and version: Chrom Latest
- Reproduction Repository: N/A

### Description:

![image](https://user-images.githubusercontent.com/43909932/166699230-83d7f3c4-fa59-465f-8251-0470624813b5.png)
![image](https://user-images.githubusercontent.com/43909932/166699349-73aa19c0-86d2-473a-ac7b-17f067f817e2.png)
![image](https://user-images.githubusercontent.com/43909932/166699408-96e18d95-584c-45e1-a310-3856e7bcd798.png)

If I change to half-width then the height matches the other cards

![image](https://user-images.githubusercontent.com/43909932/166700779-9aeed7b5-c029-4bce-bdde-45469e4c9e28.png)

**Comment by davidhemphill** [comment]
This is by design. It offers full-width cards more flexibility since they'll likely have more content.

**Comment by RhysLees** [comment]
> 

@davidhemphill  Is there a way to change this? I understand the reasoning for it but also at the same time it seems out of place when all other cards match heights. Also for the sake of saving 20px is it needed? Just curious, thanks :)

**Comment by davidhemphill** [comment]
You know, we had it the other way and people complained it was limited to the height of the other smaller cards. 😅

**Comment by RhysLees** [comment]
> You know, we had it the other way and people complained it was limited to the height of the other smaller cards. 😅

@davidhemphill 🤣 guess I'm just a pain then 🤣 I just think it looks better to the eye having everything aligned height wise when they are all the same element essentially. It's nothing breaking but maybe it could be something to consider for the update you mentioned in the footer discussion about making suff more editable.

Something like `->fixedHeight()` as a modifier or `->height(Card::FIXED_HEIGHT)` and `->height(Card::DYNAMIC_HEIGHT)`

Just a thought but thanks for your hard work with nova, I've enjoyed using it for the past 3 years 😁

**Comment by davidhemphill** [comment]
I like your taste...lol. Maybe we could provide a modifier that adds the fixed height. That seems okay to me. I'll add it to my list.

Thank you for the encouragement! It really does help hearing that.

**Comment by RhysLees** [comment]
> I like your taste...lol. Maybe we could provide a modifier that adds the fixed height. That seems okay to me. I'll add it to my list.
> 
> Thank you for the encouragement! It really does help hearing that.

Awesome I really appreciate your quick response and am glad to see this is something you will consider working on.

Hope you have a good evening 😁

**Comment by davidhemphill** [comment, marked as answer]
Already added...should come out soon. You'll be able to use `fixedHeight()`, `dynamicHeight()`, and `height(Card::DYNAMIC_HEIGHT)` or `height(Card::FIXED_HEIGHT)`.

Have a good one!

**Reply by RhysLees** [reply]
Absolute Legend!!!

Thanks, you too!
